packages/edit-site/src/components/canvas-loader/style.scss

```diff
@@ -16,6 +16,10 @@
 	& > div {
 		width: 160px;
 	}
+
+	& > .edit-site-canvas-loader__progressbar {
+		width: 160px;
```

Comment:
I still think the work to remove the width still needs too be in a separate PR.

Reply:
If the work to remove the width is considered to be separate from the stabilization work, then yes. I was under the impression it was part of it as per Lena's comment [here](https://github.com/WordPress/gutenberg/pull/61062#issuecomment-2098373670). Happy to split as well 👍🏻